src/fr/layers/wms.md

```diff
@@ -69,15 +69,15 @@ Dans OpenLayers, il y a une séparation entre les couches et les sources alors q
 ## Le constructeur `ol.source.TileWMS`
 
 Le constructeur `ol.source.TileWMS` est un argument unique qui est définit par: http://openlayers.org/en/master/apidoc/ol.source.TileWMS.html.
-L'url est la `online resource` du service WMS, et `params` est un objet litéral avec les noms des paramètres et leurs valeurs. Depuis que la version par défaut du WMS est la version 1.3.0 dans OpenLayers, vous pourriez avoir besoin de renseigner une version inférieure dans les `params` si votre WMS ne supporte par le WMS 1.3.0.
```

Comment:
Maybe someone with better French skills than  me can add a note about `TILED: true`, like I did in the English version.